archivist.py

```diff
@@ -15,6 +21,7 @@
 parser.add_argument("year", type=str)
 parser.add_argument("--category", type=str, default=None)
 parser.add_argument("--challenge", type=str, default=None)
+parser.add_argument("-m", "--mode", type=str, default="overwrite", choices=["overwrite", "skip", "duplicate"], help="overwrite(default): overwrite existing files,\nskip: skip existing files,\nduplicate: rename existing files")
```

Comment:
`overwrite` probably shouldn't be the default cause then if someone has solved the challenge and has their solution in the challenge directory, the solution will get deleted. Maybe `skip` or `prompt` (where the program asks the user whether to skip or not) would be good defaults.

Reply:
Thanks for the suggestions :D. `prompt` is a good idea, I will add it to the next commit